templates/envproxy.yaml

```diff
@@ -69,6 +69,9 @@ spec:
 {{- include "coder.environments.configMapEnv" . | indent 12 }}
 {{- include "coder.devurls.hostEnv" . | indent 12 }}
 {{- include "coder.postgres.env" . | indent 12 }}
+{{- include "coder.envproxyCredsEnv" . | indent 12 }}
+            - name: CEMANAGER_ACCESS_URL
+              value: "http://cemanager:8080"
```

Comment:
This only works inside the same cluster, but will always work with the default envproxy we ship.

Reply:
Doesn't this need the cluster domain suffix appended to it?

Reply:
it does not, if you are in the same namespace this resolves correctly from my testing.

Reply:
If you're confident that it'll work for every customer than I'm OK with it. For postgres though we have been using the full domain: https://github.com/cdr/enterprise-helm/blob/849b0697001da716a505ee8910671145d87a8213/templates/_common.tpl#L27

Reply:
Yeah I'll add it to be safe then